### ISSUE TITLE
Not working after the update of Quick Share.

### ISSUE THREAD

**Issue by Aditya1349** [grishka/NearDrop#150]
Since Nearby share merged with Quick Share the app Near Drop is not working anymore. I mean it was a great app for people like us who used android and Mac both. So I request the developer to work on this and bring this fantastic app to work again like before it will be very much appreciated.

**Comment by Aditya1349** [comment]
I don't know but i am using macbook air m1 and samsung s21 fe and it is not
working for me right now and i don't know why

**Comment by grishka** [comment]
What exactly do you mean by "now working"?... 😩

**Comment by J-shw** [comment]
I have the same issue. I use M2 MBP and Pixel 8. Neardrop is running, but my phone cannot see it as a device. So I am unable to send files to it

**Comment by Aditya1349** [comment]
I mean that when i select a file and than click on quick share icon it is
not showing my macbook like it does previously with nearby share.

**Comment by grishka** [comment]
So do you **seriously** believe that the share extension should somehow magically rename itself to "Quick Share"? By the way, it's called "NearDrop" right now.

**Comment by Aditya1349** [comment]
Earlier it was Nearby share and quick share. Nearby share is for all
androids and quick share was exclusive to only Samsung phones. But google
has merged it's nearby share with samsung's quick share with an update. But
"Near Drop" was for Mac's. But now after all this process that has happened
in Android quick share won't read my Mac. That is what i am talking about.
Earlier while transferring from nearby share it was working flawlessly and
also earlier it doesn't work with Quick share but after google integrated
both of them now its not working for me when i am sharing my files through
quick share.

**Comment by Aditya1349** [comment]
Yeah I checked it and now its working one way, like from my phone to my Mac but not other way round. Don’t know whether It will work or not.

**Comment by OrenjiSan5** [comment]
It also doesn't work for me, my phone recognizes my Mac, but the app does nothing.

**Comment by FSint653** [comment]
Working one way only, Android to Mac working fine. mac to Anroid is not working at all

Settings on phone
• set to Everyone
• Bluetoot_on
• Wifi_on
• GPS_on

<img width="409" alt="Screenshot 2024-03-17 at 7 08 35 PM" src="https://github.com/grishka/NearDrop/assets/57948707/4ce11ee1-dbce-46c8-9328-ccf9769d53e6">
![Screenshot_20240317_190910_Quick Share](https://github.com/grishka/NearDrop/assets/57948707/20862118-5399-4d4b-b6fc-0910ed5acd25)

**Comment by FSint653** [comment]
Working one way only, Android to Mac working fine. mac to Anroid is not working at all

Settings on phone
• set to Everyone
• Bluetoot_on
• Wifi_on
• GPS_on


<img width="409" alt="Screenshot 2024-03-17 at 7 08 35 PM" src="https://github.com/grishka/NearDrop/assets/57948707/72f85b78-a295-4b65-8e76-c4d9fd4ad128">

![Screenshot_20240317_190910_Quick Share](https://github.com/grishka/NearDrop/assets/57948707/d67c3566-c73b-4c06-a626-67e68267fbf1)

**Comment by megaroeny** [comment]
Same here as @FSint653. I saw with the latest release, there's a Share extension to send things like a URL (from the browser) to the phone. I get the same results on my Samsung device (which uses Quick Share in place of Nearby Share now).

**Comment by grishka** [comment]
There's no point repeating "it also doesn't work for me". Yes, I know. Yes, I still have no idea what to do about it. Help is appreciated, but tens of "it's not working" comments aren't helpful to me. They just clog my inbox.

**Comment by grishka** [comment]
And, I've never seen the UI like on @FSint653's screenshot. It's clearly Samsung's UI, not Google's. The newest Samsung device I own is the Galaxy S9, and it hasn't received any updates for more than a year.

**Comment by megaroeny** [comment]
> I still have no idea what to do about it. Help is appreciated, but tens of "it's not working" comments aren't helpful to me. They just clog my inbox.

Users are just discovering this issue and reacting to it is all. It seemed like there was some confusion, so I was clarifying. Looks like we're all on the same page now. 🙂 

For context: Quick Share uses the same Nearby Share technology, but beyond that, it's over my head. I'm not an engineer, just chiming in here. Here's a brief article to give you more context: https://www.sammobile.com/news/google-nearby-share-merged-samsung-quick-share/

**Comment by grishka** [comment]
Hmm. So I reenabled Galaxy Store, seems like I can now do this:
![Screenshot_20240320_233755](https://github.com/grishka/NearDrop/assets/1478258/bec4b3d6-1ebc-4b9a-9839-364d35cae75e)

**Comment by grishka** [comment]
Okay, so after I installed that update and rebooted the phone, I did get the same UI which is clearly an amalgamation of two technologies, Nearby Share and something Samsung made themselves. The part where you can log into two accounts (Samsung account to "share with Galaxy devices" and Google account to "share with non-Galaxy devices") gives that away.

**Comment by ilobilo** [comment]
Sometimes activity ``com.google.android.gms.nearby.sharing.ReceiveSurfaceActivity`` needs to be open for quickshare to work
